regression/goto-cc-static/chain.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+goto_cc=$1
+goto_instrument=$2
+is_windows=$3
+
+to_keep=""
+if [ $# -gt 3 ]; then
+  if [[ $4 != '' ]]; then
+    to_keep="--keep-implementations $4"
+    externalize="--externalize-symbols $4"
+  fi
+fi
+
+name=${*:$#}
+name=${name%.c}
+
+if [[ "${is_windows}" == "true" ]]; then
+  "${goto_cc}" ${to_keep} "${name}.c"
+  mv "${name}.exe" "${name}.gb"
+else
```

Comment:
To provide a double-check against some driver script wrongly supplying TRUE or 1 or something, suggest `elif [[ "${is_windows}" == "false" ]]; then` and error on anything else